packages/client/hmi-client/src/components/model/tera-parameters-metadata.vue

```diff
@@ -88,21 +167,30 @@ const parameterList = computed<
 
 const matrixModalId = ref('');
 
-function updateBaseParameter(baseParameter: string, event: any) {
-	// In order to modify the base we need to do it within the model's metadata since it doesn't actually exist in the model
-	emit('update-parameter', { parameterId: baseParameter, isMetadata: true, ...event });
-	// Cascade the change to all children
+const baseOptions = ref<
```

Comment:
Nitpick: The naming reads a little bit awkward, it isn't really an "option" if you are actively changing it. Maybe something like `editingState`, `paramEditState` or something would be more appropriate?

Reply:
Sure, I'll address this in my next PR since I rearranged some things.